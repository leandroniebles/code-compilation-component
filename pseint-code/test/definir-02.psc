Proceso prueba
  definir a como logica;
  definir b como logico;
  definir c como lógica;
  definir d como lógico;
  definir e como logicas;
  definir f como logicos;
  definir g como lógicas;
  definir h como lógicos;
  a<-"lala";
  b<-"lala";
  c<-"lala";
  d<-"lala";
  e<-"lala";
  f<-"lala";
  g<-"lala";
  h<-"lala";
FinProceso