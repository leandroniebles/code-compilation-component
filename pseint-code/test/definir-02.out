=== Line 10: SynError 125
=== Line 11: SynError 125
=== Line 12: SynError 125
=== Line 13: SynError 125
=== Line 14: SynError 125
=== Line 15: SynError 125
=== Line 16: SynError 125
=== Line 17: SynError 125
